aii-opennebula/src/main/pan/quattor/aii/opennebula/schema.pan

```diff
@@ -188,6 +188,8 @@ type opennebula_vmtemplate = {
     "ignoremac" ? opennebula_ignoremac
     "graphics"  : string = 'VNC' with match (SELF, '^(VNC|SDL|SPICE)$')
     "diskcache" ? string with match(SELF, '^(default|none|writethrough|writeback|directsync|unsafe)$')
+    @{specific image mapping driver. qcow2 is not supported by Ceph storage backends}
```

Comment:
@jrha this feature is not very hard to add so I have included this in our TODO list: 
https://github.com/quattor/configuration-modules-core/issues/1014